core/src/test/java/org/elasticsearch/index/mapper/IpFieldMapperTests.java

```diff
@@ -186,7 +186,8 @@ public void testIgnoreMalformed() throws Exception {
 
     public void testIncludeInAll() throws Exception {
         String mapping = XContentFactory.jsonBuilder().startObject().startObject("type")
-                .startObject("properties").startObject("field").field("type", "ip").endObject().endObject()
+                .startObject("properties").startObject("field").field("type", "ip")
+                .field("include_in_all", true).endObject().endObject()
```

Comment:
These tests now check that overriding to true works. Can we have tests for the default, ie that by default the field is not added to _all?